conf/base.config

```diff
@@ -44,13 +42,13 @@ process {
     withLabel:process_high {
         cpus   = { check_max( 12    * task.attempt, 'cpus'    ) }
         memory = { check_max( 72.GB * task.attempt, 'memory'  ) }
-        time   = { check_max( 16.h  * task.attempt, 'time'    ) }
+        time   = { check_max( 72.h  * task.attempt, 'time'    ) }
     }
     withLabel:process_long {
         time   = { check_max( 20.h  * task.attempt, 'time'    ) }
     }
     withLabel:process_high_memory {
```

Comment:
As far as I can see, the labels `process_high_memory` and `process_long` aren't used anywhere, so perhaps they should be removed?

Reply:
If they are from the template I would just leave them tbh. It will be easier if you start adding mroe modules to the pipeline.

Reply:
Yes, they are from the template. Will leave them alone 👍